cloudant-client/src/main/java/com/cloudant/client/api/Database.java

```diff
@@ -1010,6 +1010,59 @@ public boolean contains(String id) {
     }
 
     /**
+     * Removes an attachment from the specified document.
+     * <p>The object must have the correct {@code _id} and {@code _rev} values.</p>
+     * <P>Example usage:</P>
+     * <pre>
+     * {@code
+     * //get a Bar object from the database
+     * Bar bar = db.find(Bar.class, "exampleId");
+     * String attachmentName = "example.jpg";
+     * //now remove the remote Bar
+     * Response response = db.removeAttachment(bar, attachmentName);
+     * }
+     * </pre>
+     *
+     * @param object the document to remove as an object
+     * @param attachmentName the attachment name to remove
+     * @return {@link com.cloudant.client.api.model.Response}
+     * @throws NoDocumentException If the document is not found in the database.
```

Comment:
I think `DocumentConflictException` should be included in throws list of the documentation.